Makefile

```diff
@@ -58,7 +58,10 @@ E2E_S2I_PARAMS_URL ?= https://github.com/cpuguy83/go-md2man.git
 E2E_S2I_PARAMS_REVISION ?= master
 
 # s2i end-to-end test language of choice for using correct builder image
-E2E_S2I_LANGUAGE ?= python
+E2E_S2I_LANGUAGE ?= go
+
+#Trial
+E2E_S2I_IMAGE_SCRIPTS_URL ?=                     #./s2i/bin/usr/libexec/s2i  #registry.access.redhat.com/ubi8/go-toolset:1.19.10-3:///usr/libexec/s2i
```

Comment:
Here we should have something like the following example:

```bash
E2E_S2I_LANGUAGE =? go
# s2i's "image-script-url" flag for container-file generation
E2E_S2I_IMAGE_SCRIPTS_URL ?= image:///usr/libexec/s2i
```

On the `Makefile` we should have the default value for the `--image-scripts-url` flag, as per:

```
$ s2i build --help |grep image-scripts-url
      --image-scripts-url string         Specify a URL containing the default assemble and run scripts for the builder image (default "image:///usr/libexec/s2i"
```